tools/heuristic_clear_cache.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
Can you put this in `tools/maint/..` which is where we put stuff for maintainers and/or git users.   This will also prevent this tool being shipping in emsdk, which is good.

Reply:
check